perf/src/sigverify.rs

```diff
@@ -113,6 +122,66 @@ fn verify_packet(packet: &mut Packet) {
         return;
     }
 
+    if reject_non_vote {
+        if packet_offsets.sig_len != 1 {
```

Comment:
Do we know that all voters use the same account for paying fees as they do for their vote authority? It would be twice as expensive to use different accounts, but just wanted to be sure.